src/include/parser/pg_trigger.h

```diff
@@ -0,0 +1,81 @@
+#ifndef PG_TRIGGER_H
+#define PG_TRIGGER_H
+
+/* ----------------
+ *		compiler constants for pg_trigger
+ * ----------------
+ */
+#define Natts_pg_trigger 15
```

Comment:
```
#define Natts_pg_trigger 15
#define Anum_pg_trigger_tgqual 15
```
Are these two related? Otherwise why do you choose 15 for Natts_pg_trigger? More documentation needed.